.github/workflows/pr-checklist.yml

```diff
@@ -0,0 +1,56 @@
+name: Add editorial checklist upon review
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  add_template:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Install the GH cli
```

Comment:
I don't think this is needed: https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-github-cli 

The gh cli should be available through gh actions. (is also quite a unknown repo https://github.com/ksivamuthu/actions-setup-gh-cli)